### PR TITLE
Fix hadoop load failed when enable batch delete in unique table

### PR DESCRIPTION
**Pull request by yangzhg**
## Proposed changes

Fix hadoop load failed, when enable batch delete in unique table

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/load/Load.java [hunk 1]
```diff
@@ -758,6 +764,22 @@ public static void checkAndCreateSource(Database db, DataDescription dataDescrip
...
+                     *  For batch delete table add hidden column __DORIS_DELETE_SIGN__ to columns
+                     * eg:
+                     * (A, B, C)
+                     * ->
+                     * (A, B, C) SET (__DORIS_DELETE_SIGN__ = 0)
+                     */
+                    columnToHadoopFunction.put(column.getName(), Pair.create("default_value", Lists.newArrayList("0")));
+                    ImportColumnDesc importColumnDesc = null;
+                    try {
+                        importColumnDesc = new ImportColumnDesc(column.getName(), new FunctionCallExpr("default_value", Arrays.asList(column.getDefaultValueExpr())));
```
Comment (Gabriel39) [inline fe/fe-core/src/main/java/org/apache/doris/load/Load.java:778]: I think it's better to extract `default_value ` to a common constant to decrease hard code

**Review by Gabriel39 — COMMENTED** [review]
LGTM with minor suggestion

**Review by morningman — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/load/Load.java (hunk 1 above)
Comment (yangzhg) [reply fe/fe-core/src/main/java/org/apache/doris/load/Load.java]: `default_value` is not change for a long time, and hadoop load is deprecated, this pr just for  compitable for old systems

**Pull request merged**